spec/models/comment_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+describe Comment do
+  context "Validate" do
+    it 'should have a valid factory' do
+      FactoryGirl.create(:comment)
```

Comment:
@phendych no assert statement? do proper TDD please, the test should fail before writing code. this test will NEVER fail because there is no assert statement...